olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -153,9 +153,9 @@ spec:
     # Further Information
     For documentation on OpenShift Serverless, see:
     - [Installation
-    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/serverless/administration-guide#installing-openshift-serverless)
+    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/administration-guide#installing-openshift-serverless)
```

Comment:
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/install

Reply:
I thought it was just the version things have changed then, it is more than that. :(